crates/bevy_render/src/view/mod.rs

```diff
@@ -138,14 +140,17 @@ fn prepare_view_uniforms(
     view_uniforms.uniforms.clear();
     for (entity, camera) in views.iter() {
         let projection = camera.projection;
+        let inverse_projection = projection.inverse();
         let view = camera.transform.compute_matrix();
         let inverse_view = view.inverse();
         let view_uniforms = ViewUniformOffset {
             offset: view_uniforms.uniforms.push(ViewUniform {
                 view_proj: projection * inverse_view,
+                inverse_view_proj: inverse_projection * view,
```

Comment:
inverse(AB) = inverse(B) * inverse(A)
view_proj = projection * inverse(view)
inverse(view_proj) = inverse(projection * inverse(view))
= inverse(inverse(view)) * inverse(projection)
= view * inverse(projection)

```suggestion
                inverse_view_proj: view * inverse_projection,
```

Reply:
Okay yeah, did that, it's correct now! Thanks